src/MentionsTextInput.js

```diff
@@ -611,10 +612,11 @@ export default class MentionsTextInput extends Component {
   }
 
   onContentSizeChange(event) {
+    const extraHeight = this.props.extraContentSizeHeightOffset || 0;
     const singleLineThreshold = Platform.OS == 'android' ? 12 : 0;
     const heightDifference = event.nativeEvent.contentSize.height - this.props.textInputMinHeight;
     const newHeight = event.nativeEvent.contentSize.height + 10;
-    const height = heightDifference <= singleLineThreshold ? this.props.textInputMinHeight : newHeight;
+    const height = heightDifference <= singleLineThreshold ? this.props.textInputMinHeight : newHeight + extraHeight;
```

Comment:
I suspect these calculations could be streamlined, but that's a PR for another day.

Reply:
Probably... to be honest, I'd love to have had the bandwidth to completely rebuild the mentions input and remove the dependency we have on this repo so we have better control of the code